torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2508,7 +2508,15 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                SkipInfo('TestCommon', 'test_variant_consistency_eager'),
                SkipInfo('TestGradients', 'test_inplace_grad'),
                SkipInfo('TestGradients', 'test_inplace_gradgrad'),),
-           sample_inputs_func=sample_inputs_addcmul),
+           sample_inputs_func=sample_inputs_addcmul_addcdiv),
+    OpInfo('addcdiv',
+           dtypes=floating_and_complex_types(),
+           dtypesIfCUDA=floating_and_complex_types_and(torch.float16, torch.bfloat16),
+           skips=(
+               SkipInfo('TestCommon', 'test_variant_consistency_eager'),
+               SkipInfo('TestGradients', 'test_inplace_grad'),
```

Comment:
Suggestion:

addcdiv_ doesn't have a corresponding enttry in https://github.com/pytorch/pytorch/blob/master/tools/autograd/derivatives.yaml.
So that means inplace autograd for it isn't supported.

You'd have to add `supports_inplace_autograd=False`, instead of skipping these 2 tests.
`test_variant_consistency_eager` might pass after setting `supports_inplace_autograd=False`, as without it, it also tries to compute backward of the inplace invariant, and that might be why it failed locally for you.

Reply:
Your analysis is good, @imaginary-person, but test_variant_consistency_eager will probably still fail because of the inplace broadcasting the sample inputs want. I made a comment on the previous PR in the stack. Keeping the skip for now is OK (but please add a comment, @albanD, as described in the previous PR!)